### ISSUE TITLE
Plain CSV import fails

### ISSUE THREAD

**Issue by jameslzhu** [roddhjav/pass-import#105]
I'm encountering the following issue importing a flat CSV file (not from a password manager):

```sh
$ pass import csv ./passwds.csv --cols 'title,url,login,password,comments'
 [x] Error: ./passwds.csv is already a file.
```

Trying several variants of arguments doesn't work for me either:

```sh
$ pass import csv ./passwds.csv
 [x] Error: ./passwds.csv is already a file.
$ pass import --cols 'title,url,login,password,comments' ./passwds.csv
 [x] Error: Unable to detect the manager. Please try with: pass import <manager> ./passwds.csv
$ pass import --cols 'title,url,login,password,comments' csv ./passwds.csv
 [x] Error: ./passwds.csv is already a file.
$ pimport pass csv ./passwds.csv --cols 'title,url,login,password,comments'
 [x] Error: ./passwds.csv is already a file.
$ pimport --cols 'title,url,login,password,comments' pass csv ./passwds.csv
 [x] Error: ./passwds.csv is already a file.
```

This error message originates from the GenericCSV manager:
https://github.com/roddhjav/pass-import/blob/6cee3fac46ef53b5b6b31de26cadf5bc5d3935ab/pass_import/managers/csv.py#L105

If I were to guess the issue, it seems that it's trying to write to the CSV file instead of reading from it. Adding a `-f` flag causes the file to become completely overwritten with a blank file:

```sh
$ pass import --cols 'title,url,login,password,comments' csv ./passwds.csv -f
  w  not readable
 [x] Error: ./passwds.csv is not a valid exported csv file.
$ cat passwds.csv


```

In addition there don't appear to be any import tests for plain CSVs, only export tests. This may be why this wasn't caught in the tests.

Versions:

```
pass: v1.7.3
pass import 3.0
python 3.8.3
bash 5.0.17(1)-release
```

**Comment by roddhjav** [comment]
Hi,

This bug has been fixed with dc631144a14875b6f8c0d475bb3c14398ad1c87a. Could you try it using the master branch?

The tests for generic CSV import are here:
https://github.com/roddhjav/pass-import/blob/6cee3fac46ef53b5b6b31de26cadf5bc5d3935ab/tests/imports/test_parse.py#L65

**Comment by jameslzhu** [comment]
Whoops, didn't see that file. Import does appear to work on the latest commit.
Thanks!
